README.md

```diff
@@ -1494,7 +1494,35 @@ _You can enable the following settings in Xcode by running [this script](resourc
 
   </details>
 
-* <a id='limit-vertical-whitespace'></a>(<a href='#limit-vertical-whitespace'>link</a>) **Limit empty vertical whitespace to one line.** Favor the following formatting guidelines over whitespace of varying heights to divide files into logical groupings. [![SwiftLint: vertical_whitespace](https://img.shields.io/badge/SwiftLint-vertical__whitespace-007A87.svg)](https://github.com/realm/SwiftLint/blob/master/Rules.md#vertical-whitespace)
+* <a id='limit-consecutive-whitespace'></a>(<a href='#limit-consecutive-whitespace'>link</a>) **Limit consecutive whitespace to one blank line or space (excluding indentation).** Favor the following formatting guidelines over whitespace of varying heights to divide files into logical groupings. [![SwiftLint: vertical_whitespace](https://img.shields.io/badge/SwiftLint-vertical__whitespace-007A87.svg)](https://github.com/realm/SwiftLint/blob/master/Rules.md#vertical-whitespace) [![SwiftFormat: consecutiveSpaces](https://img.shields.io/badge/SwiftFormat-consecutiveSpaces-008489.svg)](https://github.com/nicklockwood/SwiftFormat/blob/master/Rules.md#consecutiveSpaces)
+
+  <details>
+
+  ```swift
+  // WRONG
+  struct Planet {
+
+    let mass:       Double
+    let satellites: [Planet]
```

Comment:
The nerd in me wants to point out that satellites are actually not planets. But I won't do it.

Reply:
hahahaha I had a feeling somebody would mention this 😝 

I can change this to literally anything else. Any suggestions?

Reply:
```suggestion
    let nearestNeighbor: Planet
```
?

Reply:
Defining the "nearest" planet comes with some (unintuitive!) ambiguities 😜 https://youtu.be/SumDHcnCRuU

Maybe `var hasAtmosphere: Bool`?

Reply:
Haha yeah as I think about it more this makes sense. `hasAtmosphere` sounds good.